linode/resource_linode_object_storage_key.go

```diff
@@ -57,6 +91,13 @@ func resourceLinodeObjectStorageKeyCreate(d *schema.ResourceData, meta interface
 	// secret_key only available on creation
 	d.Set("secret_key", objectStorageKey.SecretKey)
 
+	d.Set("limited", objectStorageKey.Limited)
+
+	bucketAccess := flattenLinodeObjectStorageKeyBucketAccess(objectStorageKey.BucketAccess)
+	if bucketAccess != nil {
+		d.Set("bucket_access", *bucketAccess)
```

Comment:
This is the cause of build failures. `(*schema.ResourceData).Set` handles pointer values for you.
See: https://github.com/bflad/tfproviderlint/tree/master/passes/R002
Also: https://github.com/hashicorp/terraform-plugin-sdk/blob/c31bf17bbcf68029d5174cf767c5c3e7ca3a09cd/helper/schema/resource_data.go#L172